README.md

```diff
@@ -118,6 +116,11 @@ The ``--suffixes`` option will configure a comma separated list of valid source
 The ``--whitelist`` option will only process the files listed in the file specified. This is useful for incremental analysis.
 
 The ``--xml-output`` option will generate an report in an Xml format to the path specified by the option.
+
+The ``--checkstyle-output`` option will generate a report in checkstyle xml format to the path specified by the option.
+
+The ``--extensions`` option lets you extend the code analysis. The provided extensions must be comma separated.
+
 **By default it analyses conditions, return statements, and switch cases.**
```

Comment:
This line "**By default it analyses ~", was seemed to intended to add after `--extensions` . So I move "The ``--extensions`` ~" line.

https://github.com/povils/phpmnd/commit/ce6fe2ea7de3a3cae7a230f15bdad270d2f69e96